docs_src/microservices/application/AppFunctionContextAPI.md

```diff
@@ -37,21 +37,31 @@ type AppFunctionContext interface {
 
 ## Response Data
 
-### SetResponseData()
-`SetResponseData(data []byte)` sets the response data that will be returned to the trigger when pipeline execution is complete.
+### SetResponseData
+`SetResponseData(data []byte)` 
 
-### ResponseData()
-`ResponseData()` returns the data that will be returned to the trigger when pipeline execution is complete.
+This method sets the response data that will be returned to the trigger when pipeline execution is complete.
```

Comment:
Since we are trying to be consistent across the two API docs, all thes should start with `This API` rather than `This method`